romancal/stpipe/integration.py

```diff
@@ -29,6 +29,7 @@ def get_steps():
         ("romancal.step.LinearityStep", None, False),
         ("romancal.step.PhotomStep", None, False),
         ("romancal.step.RampFitStep", None, False),
+        ("romancal.step.RefPixStep", None, False),
```

Comment:
@ddavis-stsci I thought we can setup aliases for steps here. Is there a general issue with step aliases?